lighthouse-core/gather/computed/critical-request-chains.js

```diff
@@ -22,6 +22,9 @@ class CriticalRequestChains extends ComputedArtifact {
   static isCritical(request) {
     const resourceTypeCategory = request._resourceType && request._resourceType._category;
 
+    // Iframes are considered High Priority but they are not render blocking
+    const isIframe = resourceTypeCategory === WebInspector.resourceTypes.Document._category
```

Comment:
is there an actual initiator type we can look at? what about frameId would that help us?

Reply:
parse or script I guess. Main document is set to other. What does frameid means?

Reply:
requests have a frameId associated with them, so I'm wondering if we can use that to check if the request was for a frame other than the main frame

https://chromedevtools.github.io/devtools-protocol/tot/Network/